build.csx

```diff
@@ -12,7 +14,14 @@ var versionInfoFile = "./src/VersionInfo.cs";
 var packagesDirectory = Path.GetFullPath("packages");
 var repoUrl = "https://github.com/FakeItEasy/FakeItEasy";
 var coverityProjectUrl = "https://scan.coverity.com/builds?project=FakeItEasy%2FFakeItEasy";
-var nuspecs = Directory.GetFiles("./src", "*.nuspec");
+var nuspecs = new[] {
+    "./src/FakeItEasy.nuspec",
+    "./src/FakeItEasy.Analyzer.nuspec",
+};

```

Comment:
This is because `FakeItEasy.nuspec` doesn't exist at this point. We could also evaluate it lazily, but if FakeItEasy.nuspec doesn't exist, it will just be ignored, so I thought it would be better to specify the files explicitly.